### PR TITLE
Cust 2180 redirect to apps

### PR DESCRIPTION
**Pull request by darushka**
This PR solves the redirect issue form alerts in Usermenu as well as alerts in Alert Modal.
https://whentomanage.atlassian.net/browse/CUST-2180

### CODE REVIEW

File: src/controllers/alert/alert.controller.js [hunk 1]
```diff
@@ -250,6 +250,19 @@ class AlertController {
...
   }
 
+  openApp(userAlertEvent) {
+    this.cancel();
+
+    let key = this.appById[this.accountAlertById[userAlertEvent.alertEvent.alert_id].app_id].key;
+    if(key) {
+      let path = '/accounts/' + this.accountId + '/apps/' + key;
+
+      if(this.$window.location.pathname != path) {
```
Comment (jsmylnycky) [inline src/controllers/alert/alert.controller.js]: Shouldn't this be !== ?

Comment (darushka) [inline src/controllers/alert/alert.controller.js]: @jsmylnycky 
fixed

**Comment by rynomars** [comment]
:+1:

**Comment by jsmylnycky** [comment]
:+1: Merge squash

**Comment by darushka** [comment]
Merged squash https://github.com/peachworks/ngPeach/commit/75c8a81be000e3f48ae3b6d219f5992091abb778

**Comment by jsmylnycky** [comment]
Released v0.7.9

**Pull request closed without merging**